app/models/course/virtual_classroom.rb

```diff
@@ -1,9 +1,21 @@
 # frozen_string_literal: true
 class Course::VirtualClassroom < ActiveRecord::Base
+  attr_writer :duration
```

Comment:
Would you prefer to instead do a database migration to change the `end_at` column to `duration` instead?